src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/SlimefunGuideListener.java

```diff
@@ -62,9 +62,14 @@ else if (openGuide(e, SlimefunGuideLayout.CHEST) == Result.ALLOW) {
             }
         }
         else if (openGuide(e, SlimefunGuideLayout.CHEAT_SHEET) == Result.ALLOW) {
-            // We rather just run the command here,
-            // all necessary permission checks will be handled there.
-            p.chat("/sf cheat");
+            if (!p.isSneaking()) {
```

Comment:
In all other variants is `p.isSneaking()` not negated.
I think we should keep it uniform and swap it around here too to avoid confusion here.